### PR TITLE
fix clip boxes

### PR DESCRIPTION
**Pull request by fsx950223**

### CODE REVIEW

File: efficientdet/dataloader.py [hunk 1]
```diff
@@ -237,6 +240,7 @@ def __call__(self, params):
     anchor_labeler = anchors.AnchorLabeler(input_anchors, params['num_classes'])
     example_decoder = tf_example_decoder.TfExampleDecoder()
```
Comment (mingxingtan) [inline efficientdet/dataloader.py:242]: What does this tf.autograph.experimental.do_not_convert do?

Comment (fsx950223) [reply efficientdet/dataloader.py:242]: I could debug the code.

Comment (mingxingtan) [reply efficientdet/dataloader.py:242]: would it be slower?

Comment (fsx950223) [reply efficientdet/dataloader.py:242]: > would it be slower?

No.

**Pull request merged**